### DISCUSSION TITLE
download doesn't work

### DISCUSSION THREAD

**Discussion by flyingmaiky** [gopro/labs#1157] [General]
Hello all, 
Earlier i tried downloading the gorpro labs file for gopro hero 11. 
However, the download doesn't start. DNS_PROBE_FINISHED_NXDOMAIN. 
This code keeps showing up. 

Can someone steer me in the right direction?

**Comment by dnewman-gpsw** [comment]
It is working here https://bit.ly/LABS_H11_2_30_70 (try this first) which maps to https://github.com/gopro/labs/raw/master/docs/firmware/lfs/LABS_HERO11_02_30_70.zip

Which is not working?

**Reply by epoinescu** [reply]
This link throws ERR_CONNECTION_RESET error on my PC.

**Comment by sabonev** [comment]
links for download on the site https://gopro.github.io/labs/ do not work!!!! how difficult it is guys?!?!

**Reply by dnewman-gpsw** [reply]
Yes they do, I tested just yesterday, and I'm seeing no drop-off in downloads.
![image](https://github.com/user-attachments/assets/00a06dd7-d394-4f75-ba46-2709f4b2717a)

bit.ly (the URL shortener) has made a change, so you might a have a browser issue (try something different.)

**Reply by sabonev** [reply]
obviously i tried, that is why i wrote there is an issue!!! tried with 3 different browsers - chrome, vivldi, edge! all result in ERR_CONNECTION_RESET. found the gopro 12 v2.20.70 it in some sketchy site where i could download it without any issues, which proved the problem has to be on your side....probably you don't had issues, but since yesterday i could not download no version of the gopro labs firmware, 12, 13 or whatever i choose throw the same error!!!
![image](https://github.com/user-attachments/assets/bff62772-68f5-41d8-bf8f-e2d44936199c)

**Reply by dnewman-gpsw** [reply]
Must be something at your network end, and the user downloads are continuing (as the above graph shows.)  

BTW: nothing links to https://media.githubusercontent.com/media/gopro/labs/raw/master/docs/firmware/lfs/LABS_HERO12_02_20_70.zip

The direct address is https://github.com/gopro/labs/raw/master/docs/firmware/lfs/LABS_HERO12_02_20_70.zip  It is just a page on the github, is you see this forum, you can get the firmware, as it is the same repo.  If you click on the "Code" tab above, it is possible to find all the firmware.

**Reply by sabonev** [reply]
yes but strangely enough i can access this site, but get the connection reset when i try to download. with all of my browsers is exactly the same....
it appears it is a dns issue, had to change my dns settings in order to fix it.  so it might be my providers dns settings and the most strange part is i had issues downloading only from this site....

**Reply by Riebli** [reply]
Good day 
Unfortunately, as described above, I can't download any data, see picture.
I haven't tried that with en DNS yet.
Ev is dealing with the «This website does not have a certificate»
Greetings from Switzerland

![image](https://github.com/user-attachments/assets/8c4a1d46-2a8d-4222-ae26-aa49d4e2016d)

**Reply by sabonev** [reply]
@dnewman-gpsw i am also in Switzerland, so i guess you DEFINITELY have an issue with the DNS, please fix it...

**Reply by dnewman-gpsw** [reply]
The firmware file are on github.com, same as this forum. So you can see here you can directly download. However the URL shorter is on bit.ly, and they all point here. Done that for years. I agree there is an invalid DNS entry, yet we control neither bit.ly or github.com, so there is nothing we can change. Try a different DNS.

I believe this is a local country issue, as I'm seeing a lot of attempts from Switzerland (currently the number two country for HERO13 Labs downloads, so that is a tad bizarre, as it is not normally in the top 10.) 
![image](https://github.com/user-attachments/assets/5604761a-1cd9-4fef-9cd3-448aa455e17e)

So bit.ly is tracking the forwarding, but the DNS is screwing up where to forward.

**Reply by epoinescu** [reply]
NJ, USA, same issue here, no files are accessible. Could you reach out to Github to try to resolve?
Chrome: ERR_SSL_PROTOCOL_ERROR
IE: ERR_SSL_PROTOCOL_ERROR
Oh what in the world, it works on Chrome on my phone...

**Reply by dnewman-gpsw** [reply]
It is not github, it is likely your ISP not liking the bit.ly redirects. You can always manually download from this repo, the bit.ly just points to the code here.

**Comment by Sub20Software** [comment]
I am also unable to download any of the Firmware updates. I have QR codes I want to use on my GoPro Hero12 Black
![image](https://github.com/user-attachments/assets/ca36906c-d301-499d-a6b7-dcc3ad2cd59e)
Cannot establish a secure connection to the server. 
Is there an alternate download site?

**Reply by epoinescu** [reply]
I was able to download everything on Android/ Chrome for some reason. Email me if that doesn't work and I can send you the file. 
epoinescu a gmail

**Reply by Sub20Software** [reply]
Please email the file if possible. Thank you
I tried on Android \ Chrome browser. Same error message.

**Reply by epoinescu** [reply]
My email is in previous message, or you post yours here

**Reply by Sub20Software** [reply]
support @ sub20software.com

**Comment by dnewman-gpsw** [comment]
As I explained earlier in this thread, you can download the firmware directly on this GitHub forum. So if you can see these posts, you can see the firmware. You don't have to use the bit.ly links.

**Reply by Riebli** [reply]
Unfortunately it is not from this site to download the file, see video
https://github.com/user-attachments/assets/f64d0815-cce0-4a84-9fa7-6c79f6950de2

**Reply by dnewman-gpsw** [reply]
Thank you for showing that.  Here in the US, the direct download link is https://github.com/gopro/labs/raw/refs/heads/master/docs/firmware/lfs/LABS_HERO13_01_20_70.zip?download= not https://media.githubusercontent.com/media/gopro/labs/raw/master/docs/firmware/lfs/LABS_HERO13_01_20_70.zip  I found no documentation no why media.githubusercontent.com is being used (not something I'm setting.)

**Reply by dnewman-gpsw** [reply]
HERO13 and HERO12, are now hosted off github to see if that helps.
Try either from https://gopro.github.io/labs/ again

**Reply by Riebli** [reply]
Yes, now it will be downloaded again.
Of course only Hero 12 + 13
Thank you very much

**Reply by dnewman-gpsw** [reply]
I moved only 12 & 13, as they have the most downloaded, and this appears to be a regional issue. Also it is something new, as there files have been hosted the same way for 2 years.  Where are you located?

**Reply by Riebli** [reply]
Hello, I'm in Switzerland
It worked without any problems in the beginning when the Hero 13 v1.20.70 software was released, October 25, 2024.
I don't know when it stopped working, something was certainly changed.
I also have absolutely no problem with other downloads.
My provider Swisscom
DNS 1 = 195.186.4.162
DNS 2 = 195.186.1.162
Greeting

**Comment by baugsburger** [comment]
I was having the exact same problem as everyone else, Hero11 download wouldn't work, it was driving me crazy.  I tried the download in a "private window" in Safari and it finally worked.

**Reply by dnewman-gpsw** [reply]
what county are you downloading from?

**Reply by baugsburger** [reply]
USA

**Reply by dnewman-gpsw** [reply]
That is a first. Everyone else was in Switzerland. I moved the most popular downloads to different server (not HERO11), but you would have thought github wouldn't be bad place to store files. Apparently some browsers think a URL shortener forwarding to github.com is worrying, but to miscdata.com is not. I think github is forwarding again to githubusercontent.com, and that is freaking some tools out (this is not under our control.)
